.github/workflows/migrate.yml

```diff
@@ -0,0 +1,24 @@
+name: Migration
+on:
+  push:
+    branches:
+      - fusion-react-18-vite
+
+jobs:
+  publish-preview:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
```

Comment:
Is there a reason why we've added the timeout. Nothing wrong with, just curious.

Reply:
this is was the same configuration in preview, i did not may attention but it won't make any issue